xml/obs_supported_formats.xml

```diff
@@ -33,17 +33,27 @@
    </listitem>
    <listitem>
     <para>
-     <code>#!BuildConflicts:</code>
+       <code>#!ForceMultiVersion</code> can be used to avoid reseting the build counter reset on version update.
+           s handled automatically in most cases. Just needed when it is not possible to detect
+           r sub packages defining their own version. This can be the case when the sub packages
+           eated dynamic via external rpm macros.
```

Comment:
Hey @adrianschroeter . Sorry for not reviewing this earlier, but there are missing character at the beginning of each line here.